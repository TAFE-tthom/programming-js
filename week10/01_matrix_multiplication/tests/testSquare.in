3 3 3 3
1 2 3
3 2 1
4 5 6
9 8 7
7 8 9
6 5 4
