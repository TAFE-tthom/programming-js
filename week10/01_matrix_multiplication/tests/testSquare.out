41 39 37
47 45 43
107 102 97
